src/App.js

```diff
@@ -3,17 +3,19 @@ import { Routes, Route } from "react-router-dom";
 import "./index.css";
 
 import SignUp from "./pages/SignUp";
+import Login from "./pages/Login";
 import Dashboard from "./pages/Dashboard";
 import Topbar from "./components/Topbar";
 
 function App() {
   return (
-    <div className="bg-[#D2D2D2]">
+    <div className="bg-[#D2D2D2] w-full max-w-[fixed-width] px-[30px]">
```

Comment:
The **_fixed-width_** here should have being the container width in pixels e.g 100px, 20px etc